JackSinnott/JECL_Ducks_Roguelike
# Level loading

Hey Sorry this took so long I was trying to do much in the one branch and ended up confusing myself silly. Decided to merge the level loading now as that was all this branch was ever meant to be anyway :\

**Steps to test**
1) run the project and check that the map template is loading correctly
2) Go to game.cpp and change the levelloader(1) param in constructor to levelloader(2)
3) ensure new map gets updated.

**Known issue**
At the moment we have only the one total size of rooms as the array needs to know its bounds. For example if you add another row or col to the txt file the game will crash. I will come back and try to find a way around this at a later date. even if its dirty and i just add extra constants to the globals file for different sized rooms
